Doc/library/typing.rst

```diff
@@ -2551,15 +2551,20 @@ types.
 
    This functional syntax allows defining keys which are not valid
    :ref:`identifiers <identifiers>`, for example because they are
-   keywords or contain hyphens::
+   keywords or contain hyphens, or when key names must not be mangled
```

Comment:
Could we add a cross-reference here to docs about mangling? You can use ```:ref:`private-name-mangling` ``` for this I believe (refers to a section in `reference/expressions.rst`).

Reply:
Done!